docs/developers/Centaur.md

```diff
@@ -124,28 +124,23 @@ Note: To exclude a specific test from a test suite, find the `.sh` file that run
-| Horicromtal<br>Engine Upgrade | `papi_v2beta_application.conf`**                                 | `centaur_application_`<br>`horicromtal_no_assert.conf` |
-| PAPI Upgrade                  | `papi_v1_v2alpha1_upgrade_application.conf`**                    | `centaur_application.conf`*                            |
-| Papi Upgrade<br>New Workflows | `(backend)_application.conf`                                     | `centaur_application.conf`*                            |
-| WDL Upgrade                   | `(backend)_application.conf`                                     | `centaur_application.conf`*                            |
-| (other)                       | `(backend)_application.conf`                                     | `centaur_application.conf`*                            |
+| CI Test Type                   | Cromwell Config                               | Centaur Config                                         |
+|--------------------------------|-----------------------------------------------|--------------------------------------------------------|
+| Engine Upgrade                 | `(backend)_application.conf`                  | `centaur_application.conf`*                            |
+| Horicromtal                    | `gcp_batch_horicromtal_application.conf`**    | `centaur_application_`<br>`horicromtal.conf`           |
+| Batch Upgrade<br>New Workflows | `(backend)_application.conf`                  | `centaur_application.conf`*                            |
+| WDL Upgrade                    | `(backend)_application.conf`                  | `centaur_application.conf`*                            |
+| (other)                        | `(backend)_application.conf`                  | `centaur_application.conf`*                            |
 
 | CI Test Type                  | ScalaTest Spec              | Test Directory                      |
 |-------------------------------|-----------------------------|-------------------------------------|
 | Engine Upgrade                | `EngineUpgradeTestCaseSpec` | `engineUpgradeTestCases`            |
 | Horicromtal                   | `CentaurTestSuite`          | `standardTestCases`***              |
 | Horicromtal<br>Engine Upgrade | `EngineUpgradeTestCaseSpec` | `engineUpgradeTestCases`***         |
-| PAPI Upgrade                  | `PapiUpgradeTestCaseSpec`   | `papiUpgradeTestCases`              |
-| PAPI Upgrade<br>New Workflows | `CentaurTestSuite`          | `papiUpgradeNewWorkflowsTestCases`  |
 | (other)                       | `CentaurTestSuite`          | `standardTestCases`                 |
 
 <small>
-\* Centaur Config always uses `centaur_application.conf` except when overridden with `papi_v2alpha1_centaur_application.conf`
-or `papi_v2beta_centaur_application.conf`
+\* Centaur Config always uses `centaur_application.conf` except when overridden with `gcp_batch_centaur_application.conf`
```

Comment:
Is this accurate? I'm not sure this file exists.

Reply:
good catch! switched this the the horicromtal_application.conf